### PR TITLE
SWC-6120

### PR DESCRIPTION
**Pull request by yjyoo773**

### CODE REVIEW

**Comment by yjyoo773** [comment]
Screen Shot 2022-06-08 at 1 51 59 PM

File: src/lib/style/components/_governance-dashboard.scss [hunk 1]
```diff
@@ -35,12 +35,18 @@
...
   .InputPanel {
     display: grid;
     grid-template-columns: auto 265px auto;
     gap: 25px;
 
+    .form-control {
+      border: none;
+      background-color: #ebebeb;
+    }
+
```
Comment (nickgros) [inline src/lib/style/components/_governance-dashboard.scss:49]: Let's leave the general styling of the inputs alone. If possible, we should make the change in a place that generalizes across the platform (out of scope for this PR).
```suggestion
```

File: src/lib/containers/dataaccess/AccessSubmissionDashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    <div className="AccessSubmissionDashboard bootstrap-4-backport">
+      <div className="InputPanel SubmissionGrid">
+        <div>
+          <FormLabel htmlFor="arId-filter">
+            Filter by Access Requirement Name
+          </FormLabel>
+          <AccessRequirementSearchBox
+            onChange={setAccessRequirementId}
+            placeholder="Search for an Access Requirement Name "
+          />
```
Comment (nickgros) [inline src/lib/containers/dataaccess/AccessSubmissionDashboard.tsx:108]: Can you add a prop `htmlId` to AccessRequirementSearchBox as in UserSearchBoxV2?

Then use the prop here to map the label to the input
```suggestion
          <FormLabel htmlFor="arId-filter">
            Filter by Access Requirement Name
          </FormLabel>
          <AccessRequirementSearchBox
            htmlId="arId-filter"
            onChange={setAccessRequirementId}
            placeholder="Search for an Access Requirement Name "
          />
```

**Review by nickgros — APPROVED** [review]
Looks good! Can you make the two suggested changes before merging?

**Comment by jay-hodgson** [comment]
@yjyoo773 
Screen Shot 2022-06-10 at 10 34 33 AM

**Pull request merged**